atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -329,12 +329,8 @@ def adjust_config_kwargs(self):
         self.config_kwargs['koji_hub'] = koji_map['hub_url']
         self.config_kwargs['koji_root'] = koji_map['root_url']
 
-        odcs_fallback = {
-            'api_url': self.config_kwargs.get('odcs_url'),
-            'insecure': self.config_kwargs.get('odcs_insecure')
-        }
-        odcs_map = get_odcs(self.workflow, odcs_fallback)
-        self.config_kwargs['odcs_url'] = odcs_map['api_url']
+        odcs_map = get_odcs(self.workflow, {})
+        self.config_kwargs['odcs_url'] = odcs_map.get('api_url')
         self.config_kwargs['odcs_insecure'] = odcs_map.get('insecure', False)
```

Comment:
This may not be necessary anymore, I think `odcs_url` and `odcs_insecure` are no longer used by osbs-client after  https://github.com/containerbuildsystem/osbs-client/pull/927/. However, there may be more parameters that are no longer used and I'd say removing them is out of scope for this PR.